scan/context.go

```diff
@@ -88,35 +87,40 @@ func (s *Scanner) getContextFromGitURL(gitURL string) (contextDir string, err er
 }
 
 func (s *Scanner) getContextFromURL(remoteURL string) (err error) {
-	response, err := getWithStatusError(remoteURL)
-	if err != nil {
-		return errors.Wrapf(err, "unable to download remote context from %s", remoteURL)
-	}
+	attempt := 0
```

Comment:
`http.Response.Body` is a one-time stream. If we want to retry the untar step, we need to do `http.Get()` again.